ChangeLog

```diff
@@ -1,5 +1,9 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+   * Fix compilation on Windows when building shared library, by setting
+     library search path to CMAKE_CURRENT_BINARY_DIR. #1130
```

Comment:
This PR does not fix issue #1130 according to its author, but this ChangeLog entry reads as if it did.

Reply:
Yes, but I reproduced the same error, and this fix resolved my error.

Note that this PR was written before the author mentioned it doesn't fix their issue